lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -46,6 +48,9 @@ export abstract class ClientApplication {
     // Flag to indicate if in browser environment
     protected isBrowserEnvironment: boolean;
 
+    // Callback for subscribing to events
+    private eventCallbacks: Function[];
```

Comment:
And then this would be `private eventCallbacks: EventCallbackFunction[]`

Reply:
Done.